packages/core/package.json

```diff
@@ -36,6 +37,7 @@
   },
   "homepage": "https://github.com/eslint/rewrite#readme",
   "devDependencies": {
+    "@types/json-schema": "^7.0.15",
```

Comment:
Types for `json-schema` are imported directly by the `core` types. Should this be a runtime dependency?

https://github.com/eslint/rewrite/blob/156d601181deb362a2864c4d47d4e3da8609500b/packages/core/src/types.ts#L9

Reply:
Good question. I suppose because we include `@eslint/core` as a runtime dependency of `eslint`, and `@eslint/core` can't be complete without `@types/json-schema`, then it probably makes sense to make this a runtime dependency.